frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/entity/ClientInfoResponse.java

```diff
@@ -25,19 +25,24 @@
     @JsonProperty("state")
     private String state;
 
+    @JsonProperty("cookie_consent")
+    private String cookieConsent;
+
     public ClientInfoResponse(
```

Comment:
i don't think this needs returning back in the response and its something internal

Reply:
removed